.eslintrc

```diff
@@ -4,6 +4,8 @@
 	"extends": "@ljharb",
 
 	"rules": {
-		"max-statements-per-line": [2, { "max": 2 }]
+		"max-statements-per-line": [2, { "max": 2 }],
+		"max-statements": ["error", 16],
+		"no-restricted-syntax": ["warn", "ForInStatement"]
```

Comment:
for this rule, please leave it at 2, and manually define the entire list with "ForInStatement" removed.